stable/prometheus-blackbox-exporter/templates/ingress.yaml

```diff
@@ -22,7 +22,7 @@ spec:
     - host: {{ $host }}
       http:
         paths:
-          - path: /
+          - path: {{ .Values.ingress.path }}
```

Comment:
We are in a loop, where `.` is `.Values.ingress.hosts`, so you are looking for `.Values.ingress.hosts.ingress.path`, have you tested those changes?

Reply:
Sorry, i test without ingress enabled...
I added a ci test to check that